src/charm.py

```diff
@@ -148,6 +153,23 @@ def __init__(self, *args):
         # Grafana
         self._grafana_dashboards = GrafanaDashboardProvider(self, relation_name="grafana-dashboard")
 
+        # Handle Ingress (Traefik)
```

Comment:
should we consider adding a check to ensure only one ingress integration (nginx or traefik)? i suppose it is possible to have 2 ingress services proxying to the temporal frontend, albeit a bit unrealistic